airbyte-webapp/src/components/StreamTestingPanel/StreamTestingPanel.tsx

```diff
@@ -0,0 +1,28 @@
+import { useReadStream } from "services/connectorBuilder/ConnectorBuilderApiService";
+import { useConnectorBuilderState } from "services/connectorBuilder/ConnectorBuilderStateService";
+
+import { ResultDisplay } from "./ResultDisplay";
+import { StreamSelector } from "./StreamSelector";
+import styles from "./StreamTestingPanel.module.scss";
+import { TestControls } from "./TestControls";
+
+export const StreamTestingPanel: React.FC<unknown> = () => {
+  const { jsonManifest, selectedStream, configJson } = useConnectorBuilderState();
+  const { data: streamReadData, refetch: readStream } = useReadStream({
+    manifest: jsonManifest,
```

Comment:
FYI: "manifest" is the term we are now using for the low-code YAML connector definition file, as discussed in this slack thread: https://airbytehq-team.slack.com/archives/C03TP6W8081/p1666214768391589